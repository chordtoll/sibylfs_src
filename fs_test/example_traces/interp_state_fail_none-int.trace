@type script
Pid 2 -> destroy
