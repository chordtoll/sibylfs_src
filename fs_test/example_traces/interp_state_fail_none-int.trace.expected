# processing file 'example_traces/interp_state_fail_none-int.trace' ...
        @type trace
     2: Pid 2 -> destroy
# 
# Fatal error:      2: Pid 2 -> destroy
#    no result states
# processing file 'example_traces/interp_state_fail_none-int.trace' failed
